spinalcordtoolbox/scripts/sct_label_utils.py

```diff
@@ -122,8 +122,10 @@ def get_parser():
     func_group.add_argument(
         '-disc',
         metavar=Metavar.file,
-        help="Project disc labels (-disc) onto a spinal cord segmentation (-i) to create a labeled segmentation. "
-             "Note: Unlike 'sct_label_vertebrae -discfile', this function does not involve cord straightening. "
+        help="Project disc labels ('-disc') onto a spinal cord segmentation ('-i') to create a labeled segmentation.\n"
```

Comment:
A suggestion based on the 2024-06-27 SCT dev meeting

```suggestion
        help="Project disc labels ('-disc') onto a spinal cord centerline within the axial plane to create a labeled segmentation. The centerline is automatically obtained from the provided spinal cord segmentation ('-i').\n"
```

Reply:
We should not mention the centerline, the spinal cord segmentation is used directly.